object.js

```diff
@@ -0,0 +1,41 @@
+let favorite = {
+    title:"Batman",
+    year:2014,
+    director:"Nolan",
+    actors:"bale",
+    rating:9.0,
+    timeWatched:2,
+
+    recommend:function () {
+        console.log("I recommend the film");
+    },
+    watch:function () {
+        this.timeWatched++
+    },
+    setRating:function (newRating) {
+        this.rating==newRating
+    },
+    addActor:function (actorName) {
+        this.actors=this.actors 
```

Comment:
Here, you're assigning `this.actors` to `this.actors`, which is incorrect. You need to add or append the `newName` to the actors' list. There are several ways to do this, one of which is using the `concat` method:
`this.actors = this.actors.concat(", ", actorName);`
or the simplest way is to use the plus sign
`    this.actors = this.actors + "," + actorName;
`